aggregate/housing_security/rent_median.py

```diff
@@ -0,0 +1,50 @@
+"""this chops up into six indicators they are """
+from typing import final
+import pandas as pd
+from aggregate.clean_aggregated import rename_col
+from utils.PUMA_helpers import clean_PUMAs, borough_name_mapper, get_all_boroughs, get_all_NYC_PUMAs
+from utils.dcp_population_excel_helpers import race_suffix_mapper, stat_suffix_mapper_md
+from internal_review.set_internal_review_file import set_internal_review_files
+from aggregate.load_aggregated import load_clean_pop_data
+
+year_mapper = {"12": "0812", "19": "1519"}
+
+
+def rent_median(geography: str, write_to_internal_review=False) -> pd.DataFrame:
+
+    name_mapper = {
+        "MdGR": "rent_median"
+    }
+
+    ind_name_str = "|".join([k for k in name_mapper.keys()])
+
+    clean_data = load_clean_pop_data(ind_name_str)
+
+    if geography == "citywide":
+        final = clean_data.loc[clean_data["Geog"] == "citywide"].rename(columns={"Geog": "citywide"}).copy()
+    elif geography == "borough":
+        boros = ["BX", "BK", "MN", "QN", "SI"]
+        clean_data["Geog"] = clean_data["Geog"].map(borough_name_mapper, na_action="ignore")
+        final = (
+            clean_data.loc[clean_data["Geog"].isin(boros)]
+            .rename(columns={"Geog": "borough"}).copy()
+        )
```

Comment:
Eventually we should put this into `utils/dcp_population_excel_helpers.py` - we use this all the time

Reply:
yep